indico/modules/auth/controllers.py

```diff
@@ -60,6 +61,7 @@ class RHLogin(RH):
 
     def _process(self):
         login_reason = session.pop('login_reason', None)
+        local_registration_enabled = HelperMaKaCInfo.getMaKaCInfoInstance().getAuthorisedAccountCreation()
```

Comment:
MaKaCInfo is legacy/zodb/deprecated.
We can either put it in indico.conf or in settings (via `SettingsProxy`). Probably the latter makes more sense. In that case we need a really small zodbimporter module to copy  the existing data from zodb.
Check [indico_zodbimport/modules/api.py](https://github.com/indico/indico/blob/master/indico_zodbimport/modules/api.py) on a good example on how to do it.